Conversions/IntToRoman.java

```diff
@@ -0,0 +1,101 @@
+public class IntToRoman {
+    
+    	int value(int toPret){

```

Comment:
Please have a look on the [code style conventions](http://www.oracle.com/technetwork/java/codeconventions-150003.pdf)  Between methods and ```{``` a space.